check_limits.py

```diff
@@ -1,18 +1,29 @@
 
 def battery_is_ok(temperature, soc, charge_rate):
-  if temperature < 0 or temperature > 45:
-    print('Temperature is out of range!')
-    return False
-  elif soc < 20 or soc > 80:
-    print('State of Charge is out of range!')
-    return False
-  elif charge_rate > 0.8:
-    print('Charge rate is out of range!')
-    return False
-
+  return not check_temprature(temperature)
+  return not check_state_of_charge(soc)
+  return not check_charge_rate(charge_rate)
   return True
 
+def print_text(text):
+  print(text)
+
+def check_temprature(temperature):
```

Comment:
naming of method can be improved for readability - for ex. IsTemperatureInRange(..)